extension-reference/mobile/analytics/integration.md

```diff
@@ -0,0 +1,35 @@
+```
+---
+description: >-
+  This section helps app developers understand how other modules can be integrated with Analytics.
+---
+```
+
+# Analytics Integration
+
+There are several Adobe provided extensions that can be integrated with Analytics. This document will cover the use cases on integrating with Analytics. The following integrations are all optional, except for Identity which is required by Analytics.
+
+## Acquisition
+
+**Application First Install:** The first time a user installs and opens you app the Analytics module is responsible for sending an install hit containing the referrer information from Acquisition if available.
```

Comment:
I think we can remove the acquisition section for now, since we didn't release this extension.

Reply:
We did for iOS, however not for Android. Should I still remove it?

Reply:
Yes, we did not document anything related to referrer data.